nginx/nginx.conf

```diff
@@ -0,0 +1,32 @@
+events {
+    worker_connections 1024;
+}
```

Comment:
This is not needed to be defined explicitly, default value (512) should be fine

Reply:
thanks @raunakkathuria for the insight. i removed the worker_connections key.